src/scripts/libraryMenu.js

```diff
@@ -1,6 +1,5 @@
 import escapeHtml from 'escape-html';
 import Headroom from 'headroom.js';
-
```

Comment:
I think Bill wanted them to be separated as: global deps, local deps, deps without `from` and styles.

Reply:
https://github.com/jellyfin/jellyfin-web/pull/3666/commits/3bc5b58dd83e28755d79c13dfb56719ebd0a7356